doc/source/examples/analysis/alignment_and_rms/aligning_trajectory.ipynb

```diff
@@ -49,7 +49,7 @@
     {
```

Comment:
<p>Probably will want to use a warning filter here too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Fixed.</p>